__tests__/end-to-end/jest-setup.js

```diff
@@ -1,2 +1,2 @@
-const SIMPLE_TEST_TIMEOUT = (process.env.CIRCLECI ? 20 : 10) * 1000;
+const SIMPLE_TEST_TIMEOUT = (process.env.CIRCLECI ? 30 : 10) * 1000;
```

Comment:
Why did you have to change timeout for the rename?

Reply:
Well I was fixing E2E tests anyway, and I've been thinking of upping this anyway as our E2E tests sometime fail but if you rerun them they work, and I'm not sure if it's because of too small a timeout, so did a "2 birds with one stone" thing here, sorry ;)